server/src/resolvers/mutation.ts

```diff
@@ -0,0 +1,71 @@
+import { uuid } from "uuidv4";
+import bcrypt from "bcryptjs";
+import { signupPayload, contextType, ValidationBody } from "../types";
+import { sendMail } from "../utils/sendGrid";
+import validate from "../utils/validate";
+import { createUserSchema } from "../utils/validationSchemas";
+
+const Mutation = {
+  async createUser(parent, args, { prisma, request }: contextType, info): Promise<signupPayload> {
+    const validationSchema: Array<ValidationBody> = createUserSchema(args.data);
+
+    const { errors } = validate(validationSchema);
+
+    if (Object.keys(errors).length) {
+      throw new Error("Please validate all inputs.");
```

Comment:
GraphQL-YOGA doesnt provide any out of the box error formatters?

Reply:
if not, would suggest to create error helpers such as UserInput error or Auth error. Check Apollo server for the concept.